src/pages/portal/index.js

```diff
@@ -1,9 +1,32 @@
 import React from 'react'
+import { Layout } from 'antd'
+import { Redirect, Route, Switch } from 'react-router'
 
-import GridLayout from './components/layout'
+import { Calendar, Vacations, Materials, News, Notifications } from 'features'
+import { Content } from 'common/ui-kit'
+
+import Sidebar from './components/sidebar'
 
 function PortalPage() {
-  return <GridLayout />
+  return (
+    <Layout style={{ minHeight: '100vh' }}>
+      <Sidebar />
+      <Layout>
+        <Content style={{ margin: '0 16px' }}>
+          <Switch>
+            <Route exact path="/calendar" component={Calendar} />
+            <Route exact path="/vacations" component={Vacations} />
+            <Route exact path="/materials" component={Materials} />
+            <Route exact path="/notifications" component={Notifications} />
+            <Route exact path="/" component={News} />
+            <Route default>
+              <Redirect to="/404" />
+            </Route>
```

Comment:
Also all path you can import from another constant file and reuse

Reply:
See my comment above